internal/exec/selected/selected.go

```diff
@@ -183,6 +192,28 @@ func applySelectionSet(r *Request, s *resolvable.Schema, e *resolvable.Object, s
 	return
 }
 
+func packDirectives(fe *resolvable.Field, vars map[string]interface{}) ([]directives.ResolverVisitor, error) {
+	packed := make([]directives.ResolverVisitor, len(fe.Directives))
+	for i, d := range fe.Directives {
+		args := make(map[string]interface{})
```

Comment:
Maybe we should exit early and only define this variable if the number of directive args is greater than 0.